Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -529,7 +529,7 @@ final class BuildPlanTests: XCTestCase {
     func testPackageNameFlag() throws {
         let isFlagSupportedInDriver = try DriverSupport.checkToolchainDriverFlags(flags: ["package-name"], toolchain: UserToolchain.default, fileSystem: localFileSystem)
         try fixture(name: "Miscellaneous/PackageNameFlag") { fixturePath in
-            let (stdout, _) = try executeSwiftBuild(fixturePath.appending("appPkg"), extraArgs: ["-v"])
+            let (stdout, _) = try executeSwiftBuild(fixturePath.appending("appPkg"), extraArgs: ["--vv"])
```

Comment:
This will conflict with my other change for disabling the test.

Reply:
yea we should disable the test first, then try this PR